From 944a22dea9d1ea64e7872e9fd71df1c0dde0c3cc Mon Sep 17 00:00:00 2001
From: Sebastian Lackner <sebastian@fds-team.de>
Date: Wed, 3 Jun 2015 07:20:59 +0200
Subject: ntdll/tests: Properly mark test failures with todo_wine.

Note: Those test failures are not Wine Staging specific and also
present in upstream Wine.
---
 dlls/ntdll/tests/directory.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/dlls/ntdll/tests/directory.c b/dlls/ntdll/tests/directory.c
index 21aa31f..5e1cf73 100644
--- a/dlls/ntdll/tests/directory.c
+++ b/dlls/ntdll/tests/directory.c
@@ -353,14 +353,18 @@ static void test_NtQueryDirectoryFile_case(void)
     mask.Length = mask.MaximumLength = lstrlenW(testmask) * sizeof(WCHAR);
     pNtQueryDirectoryFile(dirh, NULL, NULL, NULL, &io, data, data_size,
                           FileBothDirectoryInformation, TRUE, &mask, FALSE);
+    todo_wine
     ok(U(io).Status == STATUS_SUCCESS, "failed to query directory; status %x\n", U(io).Status);
     data_len = io.Information;
+    todo_wine
     ok(data_len >= sizeof(FILE_BOTH_DIRECTORY_INFORMATION), "not enough data in directory\n");
 
     name = dir_info->FileName;
     name_len = dir_info->FileNameLength / sizeof(WCHAR);
 
+    todo_wine
     ok(name_len == testfile_len, "unexpected filename length %u\n", name_len);
+    todo_wine
     ok(!memcmp(name, testfile_w, testfile_len * sizeof(WCHAR)), "unexpected filename %s\n",
        wine_dbgstr_wn(name, name_len));
 
-- 
2.4.2

